ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -414,6 +456,18 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity
+- as: azure-ipi-ovn-etcd-encryption-compact-arm-f28
+  cron: 21 8 27 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      OCP_ARCH: arm64
+      SIZE_VARIANT: compact
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-azure-ipi-ovn-etcd-encryption
 - as: azure-ipi-ovn-ipsec-arm-mixarch-f28
```

Comment:
Sounds like we do not have this job in source version of e2e ci.

Reply:
This is copied from openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-stable-4.15.yaml

Reply:
Is it because someone dropped the profile in e2e but forgot to dropped the corresponding upgrade job? If true, I think we can drop it.

Reply:
We are following the rule of aligning with source e2e profiles, so I think it need to be synced including 4.17 and 4.16, unless the ci jobs were added for specific reasons.

Reply:
This job included in following config files:
openshift-openshift-tests-private-release-4.14__multi-nightly-4.14-upgrade-from-stable-4.13.yaml
openshift-openshift-tests-private-release-4.15__multi-nightly-4.15-upgrade-from-stable-4.14.yaml
openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-stable-4.15.yaml

Shall we drop all of them?

Reply:
If we have plan to fill missing profiles on all version, the answer is "yes", but let us one version by one version to sync.

Reply:
Removed it from this pr